src/MusicStore/Areas/Admin/Controllers/StoreManagerController.cs

```diff
@@ -130,19 +146,19 @@ public IActionResult Create()
         // POST: /StoreManager/Edit/5
         [HttpPost]
         [ValidateAntiForgeryToken]
-        public async Task<IActionResult> Edit(Album album)
+        public async Task<IActionResult> Edit(CancellationToken requestAborted, Album album)
```

Comment:
Many actions use the default parameter value like string returnUrl = null). So, I can't use this the last parameter for all actions. For this action, I use this for the first parameter to keep consistency..but...I can make it as the last parameter for this particular action.